solutions/20.js

```diff
@@ -0,0 +1,31 @@
+// Largest Sum in an Array
+// Return the largest sum of 2 consecutive numbers given an array
+// input: [1, 2, 9, 0, 10]
+// output: 11
+
+// Zach Nagatani
+
+/**
+ * Uses a greedy approach to track the largestSum so far, updating as necessary
+ * @param {Number[]} arr - an array of numbers
+ * @returns {Number} largestSum - the largestSum of two consecutive numbers in arr
+ */
+const solution = arr => {
+    // Initialize largestSum to the sum of the first two elements in arr
+    let largestSum = arr[0] + arr[1];
```

Comment:
@zachnagatani code looks great, but please remove the comments inside of the solution such as commenting before a variable declaration or loop, etc.